int16 planeID
bool isAvoidanceWaypoint
int16 positionInQueue
---
string error
float64 latitude
float64 longitude
float64 altitude
